test-output2.txt

```diff
@@ -0,0 +1,148 @@
+?   	github.com/lazyledger/lazyledger-core/proto/tendermint/store	[no test files]
+?   	github.com/lazyledger/lazyledger-core/proto/tendermint/types	[no test files]
+?   	github.com/lazyledger/lazyledger-core/proto/tendermint/version	[no test files]
+ok  	github.com/lazyledger/lazyledger-core/proxy	(cached)
+?   	github.com/lazyledger/lazyledger-core/proxy/mocks	[no test files]
+ok  	github.com/lazyledger/lazyledger-core/rpc/client	9.836s
+?   	github.com/lazyledger/lazyledger-core/rpc/client/http	[no test files]
+?   	github.com/lazyledger/lazyledger-core/rpc/client/local	[no test files]
+ok  	github.com/lazyledger/lazyledger-core/rpc/client/mock	0.063s
+?   	github.com/lazyledger/lazyledger-core/rpc/client/mocks	[no test files]
+ok  	github.com/lazyledger/lazyledger-core/rpc/core	0.068s
+ok  	github.com/lazyledger/lazyledger-core/rpc/core/types	(cached)
+ok  	github.com/lazyledger/lazyledger-core/rpc/grpc	0.452s
+ok  	github.com/lazyledger/lazyledger-core/rpc/jsonrpc	(cached)
+ok  	github.com/lazyledger/lazyledger-core/rpc/jsonrpc/client	(cached)
+ok  	github.com/lazyledger/lazyledger-core/rpc/jsonrpc/server	(cached)
+?   	github.com/lazyledger/lazyledger-core/rpc/jsonrpc/test	[no test files]
+ok  	github.com/lazyledger/lazyledger-core/rpc/jsonrpc/types	(cached)
+?   	github.com/lazyledger/lazyledger-core/rpc/test	[no test files]
+?   	github.com/lazyledger/lazyledger-core/scripts/json2wal	[no test files]
+?   	github.com/lazyledger/lazyledger-core/scripts/wal2json	[no test files]
+ok  	github.com/lazyledger/lazyledger-core/state	2.796s
+?   	github.com/lazyledger/lazyledger-core/state/mocks	[no test files]
+ok  	github.com/lazyledger/lazyledger-core/state/txindex	(cached)
+ok  	github.com/lazyledger/lazyledger-core/state/txindex/kv	(cached)
+?   	github.com/lazyledger/lazyledger-core/state/txindex/null	[no test files]
+ok  	github.com/lazyledger/lazyledger-core/statesync	2.662s
+?   	github.com/lazyledger/lazyledger-core/statesync/mocks	[no test files]
+ok  	github.com/lazyledger/lazyledger-core/store	10.785s
+?   	github.com/lazyledger/lazyledger-core/test/app	[no test files]
+?   	github.com/lazyledger/lazyledger-core/test/e2e/app	[no test files]
+ok  	github.com/lazyledger/lazyledger-core/test/e2e/generator	(cached)
+?   	github.com/lazyledger/lazyledger-core/test/e2e/pkg	[no test files]
+?   	github.com/lazyledger/lazyledger-core/test/e2e/runner	[no test files]
+ok  	github.com/lazyledger/lazyledger-core/test/e2e/tests	(cached)
+?   	github.com/lazyledger/lazyledger-core/test/maverick/consensus	[no test files]
+?   	github.com/lazyledger/lazyledger-core/tools/tm-signer-harness	[no test files]
+ok  	github.com/lazyledger/lazyledger-core/tools/tm-signer-harness/internal	0.098s
+ok  	github.com/lazyledger/lazyledger-core/types	(cached)
+?   	github.com/lazyledger/lazyledger-core/types/consts	[no test files]
+ok  	github.com/lazyledger/lazyledger-core/types/time	(cached)
+?   	github.com/lazyledger/lazyledger-core/version	[no test files]
```

Comment:
This was probably checkedin by accident only?

Reply:
whoops. removed https://github.com/lazyledger/lazyledger-core/pull/356/commits/c11b9fe524a5125c3c287ddfc9a54b68fdb27104